requirements/optional.txt

```diff
@@ -0,0 +1,16 @@
+# some of these dependencies live on a private git server. If you have the address,
+# configure git with:
+# git config --global url.https://<private hostname>/.insteadOf https://private.repo/
+
+git+https://private.repo/CFPB/cfgov-selfregistration.git@v1.1#egg=selfregistration
```

Comment:
How do you imagine people are going to install these things? Is that documented anywhere?